presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -88,6 +91,15 @@ def test_when_analyze_with_predefined_recognizers_then_return_results(
     assert len(results) == 1
     assert_result(results[0], "CREDIT_CARD", 14, 33, max_score)
 
+def test_when_analyze_with_unsupported_language_different_then_fail(
```

Comment:
Consider adding a test with multiple inputs, e.g. mismatch, one of them is empty, null, etc. This would help ensure this logic is true in future revisions.